docs/innkrevingsoppdrag/oppdragsgiverspesifikke-valideringsregler/bronnoysundregistrene.md

```diff
@@ -10,6 +10,7 @@
 | Ugyldig periode                                                 | ugyldig-periode                                                 | 422         | `tom` kan ikke være i fremtiden. `fom` kan ikke være etter `tom`. |
 | Ugyldig stoppdato for løpende mulkt                             | ugyldig-stoppdato-for-løpende-mulkt                             | 422         | `stoppdatoForLøpendeMulkt` kan ikke være i fremtiden.             |
 | Foreldelsesfristens utgangspunkt eldre enn 2 aar og 10 maaneder | foreldelsesfristens-utgangspunkt-eldre-enn-2-aar-og-10-maaneder | 422         |                                                                   |
+| Rentebeloep er ikke tillatt for kravtype                        | rentebeloep-er-ikke-tillatt-for-kravtype                        | 422         | Rentebeløp er ikke tillatt for løpende tvangsmulkt.               |
```

Comment:
Formatet som returneres er litt annerledes, så kanskje bruke det som eksempel helt likt det man får fra endepunktet?
"Rentebeloep er ikke tillatt for kravtype=TVANGSMULKT_RETTIGHETSHAVERREGISTERLOVEN"